src/compass/s1_cslc_qa.py

```diff
@@ -261,7 +270,7 @@ def shadow_pixel_classification(self, cslc_h5py_root):
                                  pxl_qa_items)
 
 
-    def valid_pixel_percentages(self, cslc_h5py_root):
+    def valid_pixel_percentages(self, cslc_h5py_root, pol):
```

Comment:
We have another pixel percentage function for layover shadow mask i.e. `shadow_pixel_classification()` so Would like to keep the current function name to avoid confusion.

Reply:
`percent_land_and_valid_pixels`? Verbose but specific.

Reply:
@LiangJYu That looks like a good name. Changing the function name to your suggestion.